main.py

```diff
@@ -7,10 +7,40 @@
 
 class DishesCollection:
     def __init__(self):
+        self.meal_id_counter = 0
```

Comment:
this should be in a seperate Meal class, as the logic is different from dishes

Reply:
Switched to general db class.